### ISSUE TITLE
[FEATURE REQUEST] Expand IMI to other species such as CO2

### ISSUE THREAD

**Issue by msulprizio** [geoschem/integrated_methane_inversion#61]
As part of JPL's Earth Information System (EIS) there are plans to expand the IMI to include CO2 and additional observation operators.

**Comment by msulprizio** [comment]
I don't think this should be too difficult to do. To get started, you would need to:

1. Add an option for species to the config.yml file (e.g. `Species: CH4` as the default, but options would also include CO2)
2. Add new code to [these lines in template.sh](https://github.com/geoschem/integrated_methane_inversion/blob/main/src/components/template_component/template.sh#L28-L32) to create a template run directory for CO2. For example:

```
    # Commands to feed to createRunDir.sh
    # Create a GEOS-FP 0.25x0.3125 nested NA CH4 run directory by default
    # (Grid and meteorology fields will be replaced below by the settings
    #  in config.yml)
    if ( $Species == "CH4" ); then
       cmd="3\n2\n4\n4\n2\n${RunDirs}\n${runDir}\nn\n"
    elif ( $Species == "CO2" ); then
       cmd="4\n2\n4\n4\n2\n${RunDirs}\n${runDir}\nn\n"
    else
       printf "ERROR: Species ${Species} is not currently supported by the IMI"
       exit 1
    fi
```

  The values for `cmd` corresponds to the entries passed to createRunDir.sh in GCClassic.

3. Modify the scripts in the src directory  to replace CH4-specific code with options for other species. I recommend grepping the entire directory for CH4 to see where changes need to be made to allow for other species.

Also tagging @laestrada @djvaron @akurganskiy @hannahnesser for input.

**Comment by djvaron** [comment]
One more thing we'll need is a CO2 observation operator to replace the [TROPOMI_operator.py](https://github.com/geoschem/integrated_methane_inversion/blob/main/src/inversion_scripts/operators/TROPOMI_operator.py) for methane. The operator is called in [jacobian.py](https://github.com/geoschem/integrated_methane_inversion/blob/main/src/inversion_scripts/jacobian.py).

**Comment by msulprizio** [comment]
@hannahnesser is currently working on this

**Comment by msulprizio** [comment]
See draft PR #238

**Comment by msulprizio** [comment]
Initial changes for use of CO2 in the IMI has been merged into IMI 2.3.0.
